addon/index.js

```diff
@@ -145,106 +141,50 @@ function initServerEnvironmentPreference() {
     if (lastVisitedName && lastVisitedName !== currName) {
       // Switch to the last visited environment.
       simplePrefs.prefs.SERVER_ENVIRONMENT = lastVisitedName;
+      updatePrefs();
```

Comment:
Moved this call to here instead of always running at the end of this function because `updatePrefs` has already ran with the `currName` env in `main`.